.github/CHANGELOG.md

```diff
@@ -242,6 +242,9 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
 * The `qml.ResetError` is now supported for `default.mixed` device. 
   [(#1541)](https://github.com/PennyLaneAI/pennylane/pull/1541)
 
+* QNode now supports `diff_method=None`.
+  [(#1585)](https://github.com/PennyLaneAI/pennylane/pull/1585)
+
```

Comment:
Add a sentence or two here just to explain what the change enables.

Reply:
done